localisation/replace/IV_replace_l_english.yml

```diff
@@ -220,3 +225,23 @@
  Raj_12:0 "§YRecruit from their Domain§!"
  Raj_13:0 "§RConfiscation of Lands§!"
  Raj_14:0 "§YAsk for Reform Support§!"
+ # Reflavor changed ideas
+ aristo4: "Clerical Ceremonies"
+ aristo4_desc: "With a higher understanding of our faith's laws and traditions we can honor the clerics correctly during state occasions.\nEnable §MPerform a Symbolic Gesture§W Estate Interaction."
+ monarchie7: "Complex Court Customs"
+ monarchie7_desc: "Unwritten rules of our nations court etiquette compel a nobleman to arrive at court accompanied by a full retinue of at least fifty four retainers with no less than twenty four men-at-arms, ten squires, a herald, a personal cook, a houndmaster (with at least three dogs one of which has to be male), a falconer (with ...\nEnable §MEnforce Order of Precedence§W Estate Interaction."
+ republik7: "Republican Routine"
+ republik7_desc: "As a long standing republic, our government will more easily face and endure challenges.\nEnable §MGrant a non-voting Seat§W Estate Interaction."
+ assimilation3: "Citizenship Through Service" #global_manpower_modifier = 0.15
+ assimilation3_desc: "Men from newly integrated lands are incentivized to join our military force with a promise of citizenship at the end of contract."
+ assimilation4: "Theological Tolerance" #religious_unity = 0.2
+ assimilation4_desc: "We don't really mind that the unfaithful live in our dominion, as long as they stay a minority."
+ assimilation5: "Soothe Separatist Sentiment" #years_of_nationalism = -5
+ assimilation5_desc: "Why rise up and revolt against our rule, when a mutually beneficial compromise can always be achieved?"
+ assimilation6: "Community Cohesion" #reform_progress_growth = 0.1,	stability_cost_modifier = -0.15
+ assimilation6_desc: "What was up to yesterday only ours, is from today also yours. Our laws, our wealth, our institutions will serve us both and instill a sense of a shared community."
+ assimilation7: "Dissect Defunct Doctrines" #shock_damage_received = -0.025
+ assimilation7_desc: "Our new subjects may have lost their last war, but their military ideas weren't all awful. We will adapt the best of their past improvements into our own tactics."
+ expan2: "Overseas Wars" #typo
+ wehrpflicht7: "Continuous Conscription Contracts"
+ wehrpflicht7_desc: "We cannot get rid of perfectly well-trained men just because a piece of paper says they are free to go. Re-enlistment orders will come soon. I hope they won't expect a sign-on bonus.\n§GRegain manpower when disbanding units.§!\n§GVastly reduced recruitment cost, normal maintenance.§!"
```

Comment:
Why add these here instead of editing them in Idea_Variation_ideas_l_english.yml ?

Reply:
Well my practice so far was to make these changes in the replace files. The general idea is that these changes are excelbel specific, as opposed to the IV original values. Change in excelbel functionality is why these flavors needed to get changed. 
Also it's easier to see at a glance what was changed so far in the replace file than in the main.